meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor.inc

```diff
@@ -31,6 +33,13 @@ do_install() {
         -C ${S} \
         DESTDIR=${D} \
         install
+
+        install -d ${D}/${localstatedir}/lib/mender-monitor
+        # symlink /var/lib/mender-monitor to /data/mender-monitor
+        rm -rf ${D}/${localstatedir}/lib/mender-monitor
+        ln -s /data/mender-monitor ${D}/${localstatedir}/lib/mender-monitor
+
+        install -m 755 -d ${D}/data/mender-monitor
```

Comment:
You are creating the directory only to remove it again? Maybe that should be removed.

In any case, the general approach is correct. However, can you put this whole block into:

```
do_install_mender-image() {
    ...
}
```

That allows it to work for non-dual-rootfs builds as well.

Reply:
I kind of feel that it is a cunning way to create /var/lib, it is taken from the `meta-mender-core/recipes-mender/mender-client/mender-client.inc`.
moved to `do_install_mendet-image` -- thanks I had no idea about that.

Reply:
https://github.com/mendersoftware/meta-mender/pull/1448